content/lxc/getting-started.md

```diff
@@ -80,9 +80,11 @@ With that done, the last step is to create an LXC configuration file.
 
 Those values should match those found in /etc/subuid and /etc/subgid, the values above are those expected for the first user on a standard Ubuntu system.
 
-Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use the following systemd command to delegate the cgroup:
-
-    systemd-run --unit=myshell --user --scope -p "Delegate=yes" lxc-start <container-name>
+Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use a systemd command to delegate the cgroup. This part of the documention is missing here and elsewhere:
```

Comment:
Now we're back to documenting that documentation is missing. ;)
```suggestion
Running unprivileged containers as an unprivileged user only works if you delegate a cgroup in advance (the cgroup2 delegation model enforces this restriction, not liblxc). Use a systemd command to delegate the cgroup. See the following links for instructions on how to do this on different distributions:
```

Reply:
But is the required information actually included in the links for Ubuntu and Fedora?